certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -5692,6 +6269,164 @@
     }
   },
   "definitions": {
+    "AwardData": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ID.",
+          "title": "ID"
+        },
+        "name": {
+          "type": "string",
+          "description": "Name.",
+          "title": "Name"
+        },
+        "description": {
+          "type": "string",
+          "description": "Description.",
+          "title": "Description"
+        },
+        "year": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Year.",
+          "title": "Year"
+        },
+        "customerType": {
+          "type": "string",
+          "description": "Customer Type.",
+          "title": "Customer Type"
+        },
+        "allowPublicNomination": {
+          "type": "boolean",
+          "description": "Allow Public Nomination.",
+          "title": "Allow Public Nomination",
+          "enum": [
+            true,
+            false
+          ]
+        },
+        "nominationStaetDate": {
```

Comment:
@jokano12, I just noticed this typo.
Can you add a fix for this in the next PR? should be `nominationStartDate`
cc @RjImpx